src/git/pack.ml

```diff
@@ -53,6 +53,43 @@ struct
     | Tag -> Fmt.pf ppf "Tag"
 end
 
+module type ENTRY =
+sig
```

Comment:
minor nitpick: no newline needed here `module type ENTRY = sig`. This is also true in the rest of the patchset :-)